src/libs/Navigation/linkingConfig.js

```diff
@@ -7,79 +9,79 @@ export default {
                 initialRouteName: 'Report',
                 screens: {
                     // Report route
-                    Report: 'r/:reportID',
+                    Report: `${ROUTES.REPORT}/:reportID`,
```

Comment:
My concern with this is that there is route information here that is not contained within `ROUTES.js` so it's not apparent when `/:reportID` should be used and when it shouldn't. It looks to me like the only time that `ROUTE.REPORT` is referenced is when checking that path has `r` in the array (I always see it with `includes()`) so I wonder if there is a better way to do this to still maintain the functionality that is needed.

One solution that comes to mind is having:
```
// ROUTES.js
{
    REPORT: 'r',
    REPORT_WITH_ID: `r/:reportID`,
}
```

which would allow all information about the route to be contained in that file. Thoughts?

Reply:
To answer the first question.... yes! The `/:reportID` does work with `react-navigation` there's some more information here:

https://reactnavigation.org/docs/configuring-links/#passing-params

As for the second comment, you raise a good point and I think it does make sense to document these paths as constants with the parameters we expect to use with them. We might not run into too many cases where we are using `ROUTES.REPORT_WITH_ID`, but it seems like a good way to document how the routes will be used for now.

Reply:
Cool, thanks for linking that!